### PR TITLE
Nginx-based rate limiting

### PR DESCRIPTION
**Pull request by acrellin**

### CODE REVIEW

File: services/nginx/nginx.conf.template [hunk 1]
```diff
@@ -5,6 +5,19 @@ pid run/nginx.pid;
 http {
   sendfile on;
 
+  geo $limit {
```
Comment (stefanv) [inline services/nginx/nginx.conf.template:8]: This seems to mean "requests from localhost do not get rate limited".  Is that the intent?

**Review by stefanv — APPROVED** [review]

File: services/nginx/nginx.conf.template (hunk 1 above)
Comment (acrellin) [reply services/nginx/nginx.conf.template:8]: That's correct

Comment (acrellin) [reply services/nginx/nginx.conf.template:8]: Without this, `make load_demo_data` fails

Comment (acrellin) [reply services/nginx/nginx.conf.template:8]: Eventually we can template this to add Kowalski's IP address (in fritz-deploy)

**Review by kmshin1397 — APPROVED** [review]

**Pull request merged**